.pre-commit-config.yaml

```diff
@@ -19,6 +19,7 @@ repos:
   hooks:
     - id: flake8
       additional_dependencies: [flake8-docstrings]
+      exclude: ^tests/
```

Comment:
We don't want this; any specific reason you have this here?

We already have appropriate ignores setup for `tests` here:
https://github.com/sambanova/generative_data_prep/blob/zoltanc/fix_flake8/setup.cfg#L22

Reply:
Removed!